src/app/store-stats/StoreStats.m.scss

```diff
@@ -87,5 +87,13 @@
 
 .setBonuses {
   composes: flexRow from '../dim-ui/common.m.scss';
```

Comment:
Composing from flexRow may not be useful here anymore since it needs row-reverse, not sure what the best practice is.